rolling/distribution.yaml

```diff
@@ -2598,7 +2598,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/rmf_traffic-release.git
-      version: 1.3.0-2
+      version: 1.4.0-1
     source:
       type: git
       url: https://github.com/open-rmf/rmf_traffic.git
```

Comment:
@marcoag should `doc` and `source` branches change to `main`?

Reply:
Thanks for the catch, just changed them.